app/commands/beatmaps.py

```diff
@@ -142,19 +142,48 @@ async def fix_beatmapset(context: Context):
         )
 
 def update_beatmapset(session: Session, set_id: int, status: int):
-        session.query(DBBeatmapset) \
-            .filter(DBBeatmapset.id == set_id) \
-            .update({
-                'status': status,
-                'last_update': datetime.now()
-            })
-        rows_changed = session.query(DBBeatmap) \
-            .filter(DBBeatmap.set_id == set_id) \
-            .update({
-                'status': status,
-                'last_update': datetime.now()
-            })
-        return rows_changed
+    session.query(DBBeatmapset) \
+        .filter(DBBeatmapset.id == set_id) \
+        .update({
+            'status': status,
+            'last_update': datetime.now()
+        })
+    rows_changed = session.query(DBBeatmap) \
+        .filter(DBBeatmap.set_id == set_id) \
+        .update({
+            'status': status,
+            'last_update': datetime.now()
+        })
+    return rows_changed
+
+def update_beatmap(session: Session, beatmap_id: int, status: int):
+    rows_changed = session.query(DBBeatmap) \
+        .filter(DBBeatmap.id == beatmap_id) \
+        .update({
+            'status': status,
+            'last_update': datetime.now()
+        })
```

Comment:
you forgot `session.commit()` :trollface:

Reply:
It's done elsewhere